src/components/LanguagePicker/LanguagePicker.tsx

```diff
@@ -73,3 +127,9 @@ const messages = defineMessages({
     defaultMessage: '_',
   },
 })
+
+type Props = {
+  onPressConfirmButtonCallback?: () => void
```

Comment:
onPress

Reply:
That functionality has been removed but I will apply next time thanks!